spec/features/course/announcement_sticky_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe 'Announcement sticky' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.